pyracf/common/security_admin.py

```diff
@@ -84,6 +93,12 @@ def __overwrite_field_data(self, field_data: dict):
         """Overwrite field data in valid segment traits dictionary"""
         self._valid_segment_traits = field_data
 
+    def __add_more_secrets(self, new_secrets: dict):
```

Comment:
I would call this field `__add_additional_secret_traits`.

Reply:
agree, this is clearer and more descriptive. Added.